.eslintrc.json

```diff
@@ -91,6 +91,7 @@
     ],
     // Decided to turn off:
     "react/prop-types": "off",
+    "react/react-in-jsx-scope": "off",
```

Comment:
suggestion: according to [react eslint plugin docs](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/react-in-jsx-scope.md), we should instead extend `react/jsx-runtime` config:

> If you are using the new JSX transform from React 17, you should disable this rule by extending react/jsx-runtime in your eslint config (add "plugin:react/jsx-runtime" to "extends").